src/views/Cards.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+import CardsList from '../components/CardsList';
+import { getCards } from '../services/cards';
+
+export default function Cards() {
+  const [cards, setCards] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getCards();
+      setCards(data);
+      setLoading(false);
+    };
+    if (loading) {
```

Comment:
No need to have this fetch conditional upon loading. It should run once when the page first mounts